app/views/articles/index.html.erb

```diff
@@ -30,7 +30,7 @@
       data-params="<%= params.merge(sort_by: "hotness_score", sort_direction: "desc").to_json(only: %i[tag username q sort_by sort_direction]) %>" data-which="<%= @list_of %>"
       data-tag=""
       data-feed="<%= params[:timeframe] || "base-feed" %>"
-      data-articles-since="<%= Timeframe.datetime(params[:timeframe]) %>">
+      data-articles-since="<%= Timeframe.datetime_iso8601(params[:timeframe]) %>">
```

Comment:
I wonder if there is a way or a world where we could ensure that we always use iso8601 date time in places like these. This seems like a very easy thing to miss or accidentally forget to include 🙃